ios/LibTorch_vision_ops.podspec

```diff
@@ -0,0 +1,49 @@
+#
+# Be sure to run `pod lib lint LibTorch_vision_ops.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'LibTorch_vision_ops'
+  s.version          = '0.9.0'
+  s.summary          = 'libtorchvision_ops library for LibTorch'
+
+# This description is used to generate tags and improve search results.
+#   * Think: What does it do? Why did you write it? What is the focus?
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = <<-DESC
+Additional TorchVision operations for LibTorch
+                       DESC
+
+  s.homepage         = 'https://github.com/elb3k/vision'
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = 'elbekkhoshimjonov@gmail.com'
+  s.source           = { :http => "https://github.com/elb3k/vision/releases/download/v#{s.version}/libtorchvision_ops_ios_#{s.version}.zip" }
```

Comment:
Update the above information to point to the main repo

Reply:
Updated information